.github/workflows/deployment.yml

```diff
@@ -126,6 +141,8 @@ jobs:
           set -e
           cd jabgui
 
+          eval "$(mise hook-env)"
+
```

Comment:
seems like the env.JAVA_HOME var is not set correctly 
jlink failed with: Error: jlink version 25.0 does not match target java.base version 21.0

Reply:
https://github.com/jdx/mise/discussions/4973